themes/bootstrap3/templates/cart/cart.phtml

```diff
@@ -18,23 +18,28 @@
         </label>
       </div>
       <?php if ($this->userlist()->getMode() !== 'disabled'): ?>
-        <button type="submit" class="toolbar-btn btn-type-save" name="saveCart" value="1" title="<?=$this->transEscAttr('bookbag_save')?>" value="1">
+        <button type="submit" class="toolbar-btn" name="saveCart" value="1" title="<?=$this->transEscAttr('bookbag_save')?>" value="1">
+          <?=$this->icon('save') ?>
           <?=$this->transEsc('Save')?>
         </button>
       <?php endif; ?>
-      <button type="submit" class="toolbar-btn btn-type-email" name="email" value="1" title="<?=$this->transEscAttr('bookbag_email')?>" value="1">
+      <button type="submit" class="toolbar-btn" name="email" value="1" title="<?=$this->transEscAttr('bookbag_email')?>" value="1">
+        <?=$this->icon('email') ?>
         <?=$this->transEsc('Email')?>
       </button>
       <?php $exportOptions = $this->export()->getActiveFormats('bulk'); if (count($exportOptions) > 0): ?>
-        <button type="submit" class="toolbar-btn btn-type-export" name="export" value="1" title="<?=$this->transEscAttr('bookbag_export')?>" value="1">
+        <button type="submit" class="toolbar-btn" name="export" value="1" title="<?=$this->transEscAttr('bookbag_export')?>" value="1">
+          <?=$this->icon('export') ?>
           <?=$this->transEsc('Export')?>
         </button>
       <?php endif; ?>
-      <button type="submit" class="toolbar-btn btn-type-print dropdown-toggle" name="print" value="1" title="<?=$this->transEscAttr('print_selected')?>" value="1">
+      <button type="submit" class="toolbar-btn dropdown-toggle" name="print" value="1" title="<?=$this->transEscAttr('print_selected')?>" value="1">
+        <?=$this->icon('printer') ?>
```

Comment:
This icon doesn't seem to be set up correctly -- I'm not seeing a print icon in any theme.